packages/pyolite-kernel/src/declarations.d.ts

```diff
@@ -7,10 +7,3 @@ declare module '!!file-loader*' {
   const res: string;
   return res;
 }
-
```

Comment:
these deletions feel great... i wonder if this whole file can be deleted now?

Reply:
we do still need it, for now, for the file-loader stuff... but `raw` can go... as can `buildScripts`... having a look...